steps/tuist/0.1.0/step.yml

```diff
@@ -0,0 +1,46 @@
+title: tuist
```

Comment:
Based on the Step Title conventions: `Run tuist` (or something like this) would be better: https://devcenter.bitrise.io/en/steps-and-workflows/developing-your-own-bitrise-step/developing-a-new-step.html#the-title